src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -133,6 +92,33 @@ const ScenarioParameters = ({
     }
   };
 
+  useEffect(() => {
+    // Reset parameters
+    resetParameters(false);
+    // eslint-disable-next-line
+  }, []);
+
+  useEffect(() => {
+    defaultScenarioParameters.current = currentScenario.data.parametersValues;
+    /// //////////////////////////////////////////////////////////////////////// INITIAL STOCK
+    const initialStockParameter = currentScenario.data?.parametersValues?.find(el => el.parameterId === INITIAL_STOCK_PARAM_ID);
+    setInitialStockDatasetId(initialStockParameter?.value === undefined ? '' : initialStockParameter.value);
+    /// //////////////////////////////////////////////////////////////////////// INITIAL STOCK
+    // eslint-disable-next-line
+  }, [currentScenario, changeEditMode, initialStockFile.status]);
+
+  // State for bar defaultScenarioParameters
+  const [stock, setStock] = useState();
+  const [restockQuantity, setRestockQuantity] = useState();
+  const [waitersNumber, setWaitersNumber] = useState();
+
+  // State for basic input types examples defaultScenarioParameters
+  const [currency, setCurrency] = useState();
+  const [currencyName, setCurrencyName] = useState();
+  const [currencyValue, setCurrencyValue] = useState();
+  const [currencyUsed, setCurrencyUsed] = useState();
+  const [startDate, setStartDate] = useState();
+
```

Comment:
Not initializing the states on creation leads to many errors in the browser console such as "Failed prop type: The prop `stock` is marked as required in `BasicTypes`, but its value is `undefined`."

Reply:
:+1: